docs/course_materials/compound_data_types.html

```diff
@@ -189,32 +245,163 @@ <h1><b>
-
-t = (1, 2, 'hi')
-type(t) # &ltclass 'tuple'&gt
+                        # Method #1:
+                        my_tuple = tuple()
+                        print(my_tuple) # ()
+                        type(my_tuple) # &ltclass 'tuple'&gt
+
+                        # Method #2:
+                        my_tuple = ()
+                        print(my_tuple) # ()
+                        type(my_tuple) # &ltclass 'tuple'&gt
+                    </code></pre>
+                </section>
+                <section>
+                    <h3><b>Creating a tuple</b></h3>
+                    <ul>
+                        <li>Create a populated tuple</li>
+                    </ul>
+                    <pre><code data-trim data-noescape class="hljs lang-python">
+                        nums_tuple = (1, 2)
+                        print(nums_tuple) # (1, 2)
 
-t = 1, 2, 'hi' # be careful
-type(t) # &ltclass 'tuple'&gt
+                        strings_tuple = ('one', 'two')
+                        print(strings_tuple) # ('one', 'two')
 
-t = tuple([1,2,'yo']) # casting a list to a tuple
-print(t) # (1, 2, 'yo')
+                        mix_tuple = (1, 'two')
+                        print(mix_tuple) # (1, 'two')
+                    </code></pre>
+                </section>
+                <section>
+                    <h3><b>Tuple syntax</b></h3>
+                    <ul>
+                        <li>Technically, tuples are indicated by commas ','</li>
+                        <li>Thus, a tuple can be created without parentheses:</li>
+                    </ul>
+                    <pre><code data-trim data-noescape class="hljs lang-python">
+                        my_tuple = 1, 2, 'three'
+                        print(my_tuple) # (1, 2, 'three')
+                        type(my_tuple) # &ltclass 'tuple'&gt
+                    </code></pre>
+                    <ul>
+                        <li>Yet, it's adviced to wrap the tuple in parentheses '()' to improve readability</li>
+                    </ul>
+                </section>
+                <section>
+                    <h3><b>Tuple unpacking</b></h3>
+                    <ul>
+                        <li>As the name suggests, the action unpacks the values of a tuple into separate variables</li>
+                    </ul>
+                    <pre><code data-trim data-noescape class="hljs lang-python">
+                        my_tuple = (1, 2, 3)
+                        a, b, c = my_tuple
+                        print(a) # 1
+                        print(b) # 2
+                        print(c) # 3
                     </code></pre>
                 </section>
                 <section data-background-size="cover" data-background-image="media/exr_background.jpeg" data-markdown="exercises/04_Compound_data_types/class_exercise_2.md" data-separator="^\n\n\n" data-separator-vertical="^\n\n">
                 </section>
             </section>
+            <!-- Slicing -->
```

Comment:
I think the slicing should be inside the list, because their classwork and homework are about slicing.

Reply:
It's a large topic and it also works with Tuples. Should we really push it into lists?

Reply:
it's also relevant for strings. 
I think it should be inside lists because it's a big part of the list. 
and you can add one slide inside tuple. that says it works for tuple as well with some examples.